data/src/main/java/com/teamagam/gimelgimel/data/message/repository/cache/room/mappers/MessagesTextSearcherImpl.java

```diff
@@ -0,0 +1,44 @@
+package com.teamagam.gimelgimel.data.message.repository.cache.room.mappers;
+
+import com.teamagam.gimelgimel.data.message.repository.cache.room.AppDatabase;
+import com.teamagam.gimelgimel.data.message.repository.cache.room.dao.SearchMessagesDao;
+import com.teamagam.gimelgimel.data.message.repository.cache.room.entities.ChatMessageEntity;
+import com.teamagam.gimelgimel.domain.messages.MessagesTextSearcher;
+import com.teamagam.gimelgimel.domain.messages.entity.ChatMessage;
+import java.util.ArrayList;
+import java.util.List;
+import javax.inject.Inject;
+
+public class MessagesTextSearcherImpl implements MessagesTextSearcher {
+
+  private SearchMessagesDao mSearchMessagesDao;
+  private MessagesEntityMapper mMessagesEntityMapper;
+
+  @Inject
+  public MessagesTextSearcherImpl(AppDatabase appDatabase,
```

Comment:
Ask for the SearchMessageDao instead of the DB

Reply:
done